src/__tests__/index.test.js

```diff
@@ -165,6 +165,8 @@ describe('denormalize', () => {
     const user = new schema.Entity('users');
     const comment = new schema.Entity('comments', {
       user: user
+    }, {
+      idAttribute: (value, parent, key) => `${value.id}-${parent.id}-${key}`
```

Comment:
Can we make a separate test case that's specific to this?

Reply:
Sure. Done